test/fn/upload-custom-translations.spec.js

```diff
@@ -362,5 +362,13 @@ describe('upload-custom-translations', () => {
           });
       });
     });
+
+    describe('all-versions', () => {
```

Comment:
Can we get a test confirming the successful case and validating that the language name is set properly?

Reply:
Done